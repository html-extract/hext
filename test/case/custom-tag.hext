<yt-formatted-string @text:x />
