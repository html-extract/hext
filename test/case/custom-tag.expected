{"x":"This is some content in a custom tag"}
